### PR TITLE
Problem 4

### PR DESCRIPTION
**Pull request by exalt90**
Print string X times.

### CODE REVIEW

File: curriculum/js1/4.js [hunk 1]
```diff
@@ -6,8 +6,14 @@
...
 
-const solution = (a, b) => {
-  return 0;
+const solution = (x,b,str= "",i=0) => {
+  if(x === i)
+  {
+   return str;
+  };
+
+  str = b  + str
```
Comment (seemcat) [inline curriculum/js1/4.js]: you forgot the ; at the end of this line.

**Review by seemcat — CHANGES_REQUESTED** [review]

File: curriculum/js1/4.js (hunk 1 above)
Comment (exalt90) [reply curriculum/js1/4.js]: I see.

File: curriculum/js1/4.js [hunk 2]
```diff
@@ -6,8 +6,14 @@
  * @returns {string}
  */
 
-const solution = (a, b) => {
-  return 0;
+const solution = (x,b,str= "",i=0) => {
+  if(x === i)
```
Comment (songz) [inline curriculum/js1/4.js]:
```
if(...) {
  ...
}
```

File: curriculum/js1/4.js [hunk 3]
```diff
@@ -6,8 +6,14 @@
...
+  {
+   return str;
+  };
+
+  str = b  + str
+  return solution(x,b,str,i+1);
```
Comment (songz) [inline curriculum/js1/4.js]: ` return solution(x,b, str+b, i+1)`

**Review by songz — APPROVED** [review]

**Pull request merged**